app/src/main/java/nu/yona/app/utils/PreferenceConstant.java

```diff
@@ -77,4 +77,6 @@
     String PROFILE_UUID = "profileUUID";
 
     String PROFILE_OTP_STEP = "profileOtpStep";
+
+    String YONA_ENCRYPTION_METHOD = "yonaEncryptionUpgraded";
```

Comment:
The variable was renamed but the value forgotten. Please change it to:

```
String YONA_ENCRYPTION_METHOD = "yonaEncryptionMethod";
```

Reply:
Updated. Missed it some how.